data/stats/get_runway_stats.py

```diff
@@ -0,0 +1,37 @@
+""" Gather runway statistics for each 
+    airport from given airport runway data. 
+"""
+
+from dataclasses import dataclass, field
+from typing import TypedDict
+
+import pandas as pd
+
+
+class RunwayDict(TypedDict):
+    length_in_ft: int
+    length_in_m: int
+    name: str
+    surface: str
```

Comment:
Are these fields different to the existing `Runway` type?

Reply:
The items in the list are in a dictionary so I'm accessing it by key.  So if I annotate it as the existing type, mypy complains about me trying to index a dataclass

Reply:
Yeah it's just a dict really but I thought using `TypeDict `would be more specific than `dict[str,str|int]`

Reply:
Actually I think it needs the TypedDict, at least for line 33 otherwise mypy will complain about trying to do a sum on a string